src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+/* #region card */
```

Comment:
In my opinion, there is not so necessary to use region cards, because you have been already separated blocks styles by files. What do you think about it?

Reply:
agree